docs/multicluster/upgrade.md

```diff
@@ -37,6 +37,31 @@ for the feature in new version.
 It should have no impact during upgrade to those imported resources like Service, Endpoints
 or AntreaClusterNetworkPolicy.
 
+### Upgrade from a version prior to v1.13
+
+Prior to v1.13, the CRD `ClusterClaim` is used to define both Cluster and ClusterSet IDs
+for a ClusterSet. Since Antrea v1.13, the CRD `ClusterClaim` is removed, a new field `clusterID`
```

Comment:
and the `ClusterSet` CRD solely defines a ClusterSet. The name of a ClusterSet CR must match the ClusterSet ID, and a new `clusterID` field specifies the Cluster ID.